v3/plugins/redhat/vscode-didact/0.1.14/meta.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v2
+publisher: redhat
+name: vscode-didact
+version: 0.1.14
+type: VS Code extension
+displayName: YAML
+title: Didact Tutorial Tools by Red Hat
+description: Provides interactive tutorial support to Visual Studio Code, via didact links calling VS Code commands, Markdown, AsciiDoc, and the VS Code webview.
+icon: https://www.eclipse.org/che/images/logo-eclipseche.svg
+repository: https://github.com/redhat-developer/vscode-didact
+category: Other
+firstPublicationDate: "2020-05-06"
+spec:
+  extensions:
+    - https://github.com/redhat-developer/vscode-didact/releases/download/0.1.14/vscode-didact-0.1.14.vsix
```

Comment:
it is possible to use download.jboss.org https://download.jboss.org/jbosstools/vscode/stable/vscode-didact/ They are pushed automatically when releasing. It avoids having to push the artifact to github release (which is done manually as far I know)